src/main/java/be/seeseemelk/mockbukkit/MockUnsafeValues.java

```diff
@@ -1,30 +1,99 @@
 package be.seeseemelk.mockbukkit;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.Set;
-
+import com.destroystokyo.paper.util.VersionFetcher;
 import com.google.common.collect.Multimap;
-import org.bukkit.Material;
-import org.bukkit.NamespacedKey;
-import org.bukkit.UnsafeValues;
+import io.papermc.paper.inventory.ItemRarity;
+import net.kyori.adventure.text.flattener.ComponentFlattener;
+import net.kyori.adventure.text.serializer.gson.GsonComponentSerializer;
+import net.kyori.adventure.text.serializer.legacy.LegacyComponentSerializer;
+import net.kyori.adventure.text.serializer.plain.PlainComponentSerializer;
+import net.kyori.adventure.text.serializer.plain.PlainTextComponentSerializer;
+import org.bukkit.*;
 import org.bukkit.advancement.Advancement;
+import org.bukkit.attribute.Attributable;
 import org.bukkit.attribute.Attribute;
 import org.bukkit.attribute.AttributeModifier;
+import org.bukkit.block.Block;
 import org.bukkit.block.data.BlockData;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
 import org.bukkit.inventory.CreativeCategory;
 import org.bukkit.inventory.EquipmentSlot;
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.material.MaterialData;
 import org.bukkit.plugin.InvalidPluginException;
 import org.bukkit.plugin.PluginDescriptionFile;
+import org.jetbrains.annotations.NotNull;
 
-@SuppressWarnings("deprecation")
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+@Deprecated
 public class MockUnsafeValues implements UnsafeValues
 {
 
 	private final Set<String> compatibleApiVersions = new HashSet<>(Arrays.asList("1.13", "1.14", "1.15", "1.16", "1.17", "1.18"));
 
+	//Paper start
```

Comment:
Is this comment really necessary?

Reply:
It's not necessary. I just added it to keep in Style with the Paper Changes in Spigot. I can remove those Comments if needed

Reply:
Ah, okay!